spaces/S000065/properties/P000065.md

```diff
@@ -1,13 +1,10 @@
 ---
 space: S000065
-property: P000059
+property: P000065
 value: true
 refs:
 - doi: 10.1007/978-1-4612-6290-9_6
   name: Counterexamples in Topology
 ---
 
 Obviously, $|X| = \mathfrak{c}$.
```

Comment:
I suggest "By construction" instead of "Obviously". Maybe even spell out `|X|=|[0,1]\cup\{1^*\}|=\mathfrak c`.